### ISSUE TITLE
When searching cubes, right clicking in a cube should have option to open a cube

### ISSUE THREAD

**Issue by wgbusch** [dekkerglen/CubeCobra#2184]
**What would you like to be able to do?**
When I use the search function of Cube Cobra, if I right click on a cube, the option to open that cube in a new tab should appear. Right now, the only way to open a cube in the search screen is by left click, so the search gets lost. Using right click I could open multiple cubes from the same search.

**Describe how you would use the feature**
To open multiple cubes in the same search without having to re-search again.

**Additional context**

![image](https://user-images.githubusercontent.com/29581073/135267509-4ca5e00b-b9cc-4edc-a7b3-7fc89a27915a.png)

As you can see, there is no way to open a cube in a separate tab.

**Comment by dekkerglen** [comment]
Duplicate of #1689 

Currently waiting on a dependency to upgrade in order to do this
